addons/website_profile/views/website_profile.xml

```diff
@@ -644,7 +644,7 @@
             Your account does not have an email set up. Please set it up on <a class="alert-link" t-att-href="my_account_redirect">your account settings</a>.
         </div>
         <div t-elif="not validation_email_sent and not is_public_user and user.karma == 0" t-att-class="send_alert_classes" role="alert">
-            <button type="button" class="close validation_email_close" data-bs-dismiss="alert" aria-label="Close">×</button>
+            <button type="button" class="btn-close validation_email_close" data-bs-dismiss="alert" aria-label="Close"></button>
```

Comment:
Would that be an issue to close the button directly? :) (Same on the other one)

`<button type="button" ... aria-label="Close"/>`

I see that most existing ones do not close them immediately though. Probably due to the BS5 upgrade script? As it is targeting 16.0, I do not mind keeping it that way :)

Reply:
I don't think that's an issue :thinking: Found some that are immediately closed in 16.0 so it shouldn't be a problem :thinking: 
I updated my code :slightly_smiling_face: